DARPA-ASKEM/terarium
# Model to latex with mira task

# Description
Replacing sciml's `model-to-latex` with a mira task runner job.

- This means we will have `\operatorname` rather than `\mathrm` I dont think this is an issue but please let me know
- These equations are now able to have observables. I am confirming if we want to add these or not
- The test added here is following the same idea of `amr-to-mmt`'s test. AKA can the task be created.
If we want more in depth testing please let me know either here or in the `askem-server` channel 

# Follow up:
Next PR will include the remaining removal of sciml. Just trying to limit the scope of this PR for readability.

# Testing:
- Go to a model page via project side panel, or model's drilldown in workflow and see equations:
![image](https://github.com/user-attachments/assets/0222167e-c2b3-4ead-bf59-e745fa75e7d5)

- Go to a broken model page and see that it correctly errors out on the task runner rather than waiting for a timeout (no pic of logs thats kindve silly?)
Here is a sample broken model where I have changed the expression to be a bit nonsense for `t0` This will fail having a diagram as well as equations
[SIR with failed equations.json](https://github.com/user-attachments/files/17044884/SIR.with.failed.equations.json)
